Assets/Scripts/Footstep.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Footstep : MonoBehaviour {
+
+	public GameObject body;
+	BodyController bodyController;
+	AudioManager audioManager;
+	private string footOrientationString = "Right";
+	private string playerString = "P1";
+
+	void Start () {
+		bodyController = body.GetComponent<BodyController> ();
+		audioManager = FindObjectOfType<AudioManager> ();
+		if(gameObject.name.Contains("L")) {
+			footOrientationString = "Left";
+		}
+
+		GameObject playerOne = GameObject.Find ("PlayerTwo");
+		if(gameObject.transform.IsChildOf(playerOne.transform)) {
+			playerString = "P2";
+		}
```

Comment:
The naming here is confusing. We're initializing a playerOne go, but it's named "PlayerTwo" in the editor? Should `playerOne` on lines 20 and 21 be `playerTwo` instead?

Reply:
Nice catch